e2e/home.component.e2e-spec.ts

```diff
@@ -11,9 +11,11 @@ describe('Home Component', () => {
 
     // Host URL + SPA Name automatically included
     SkyHostBrowser.get('/');
-    expect(element(by.tagName('h1')).getText()).toBe('Stache 2');
 
-    // // Since we've said this is an async test, we need to call done
-    done();
+    element(by.tagName('h1')).getText()
+      .then((text: string) => {
+        expect(text).toBe('Stache 2');
+        done();
```

Comment:
This is all well and good for when we create this template... but if the users change this text in their project (as they should) won't this cause issues and failures when they try to publish?